src/components/common/form/message-item/MessageItem.spec.jsx

```diff
@@ -0,0 +1,72 @@
+import { screen } from '@testing-library/react';
+import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
+
+import MessageItem from './MessageItem';
+
+describe('MessageItem', () => {
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
+  it('should render a summary component with a single date and messages details', () => {
+    const summaryMessages = {
+      update: {
+        template: `datesMoved`,
+        variables: { startDate: '2022-09-21' },
+      },
+    };
+    renderWithTimecardContext(
+      <MessageItem
+        keys={['update']}
```

Comment:
What is this keys prop? It isn't a prop in the component

Reply:
It's used by the message summary to determine what order the messages should be displayed in if there's more than one